### PR TITLE
Add descriptions to remaining TypeScript Props

### LINKED ISSUE #11: Document props that don't have descriptions

### PR DESCRIPTION
**Pull request by Caleb-Ellis**
## Done

- Added descriptions for remaining TypeScript Props
- Converted a couple of `enum`s to `as const`s

### QA steps

- N/A.

## Fixes

Fixes #11

### CODE REVIEW

File: src/components/Button/Button.tsx [hunk 1]
```diff
@@ -3,29 +3,61 @@ import PropTypes from "prop-types";
...
+import type { ValueOf } from "../../types";
+
+export const ButtonAppearance = {
+  BASE: "base",
+  BRAND: "brand",
+  LINK: "link",
+  NEGATIVE: "negative",
+  NEUTRAL: "neutral",
+  POSITIVE: "positive",
+} as const;
```
Comment (huwshimi) [inline src/components/Button/Button.tsx:15]: Nice, thanks!

File: src/components/Button/Button.tsx [hunk 2]
```diff
@@ -3,29 +3,61 @@ import PropTypes from "prop-types";
...
 /**
  * The type of the Button props.
  * @template P - The type of the props if providing a component to `element`
  */
 export type Props<P = null> = {
-  appearance?: ButtonAppearance | string;
+  /**
+   * The appearance of the button.
+   */
+  appearance?: ValueOf<typeof ButtonAppearance>;
```
Comment (huwshimi) [inline src/components/Button/Button.tsx:25]: Does this still need `| string` to allow custom button types?

File: src/components/RadioInput/RadioInput.tsx [hunk 3]
```diff
@@ -5,6 +5,9 @@ import type { ReactNode } from "react";
 import CheckableInput from "../CheckableInput";
 
 export type Props = {
+  /**
+   * The lavel of the radio input.
```
Comment (huwshimi) [inline src/components/RadioInput/RadioInput.tsx:9]: A lovely lavel?

**Review by huwshimi — APPROVED** [review]
Thanks for all this! A couple of small comments.

File: src/components/RadioInput/RadioInput.tsx (hunk 3 above)
Comment (Caleb-Ellis) [reply src/components/RadioInput/RadioInput.tsx]: The loveliest

**Pull request merged**